### PR TITLE
data: rattandeep1998

### PR DESCRIPTION
**Pull request by rattandeep1998**
## Changes proposed

Added data for @rattandeep1998 LinkFree profile.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2889"></a>

### CODE REVIEW

**Review by shivaabhishek07 — APPROVED** [review]

File: data/rattandeep1998.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow my Open Source journey",
+      "url": "https://github.com/rattandeep1998",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/pradumnasaraf",
```
Comment (senali-d) [inline data/rattandeep1998.json:26]: Correct the Linked in url
```suggestion
      "url": "https://www.linkedin.com/in/rattandeep-singh",
```

**Review by senali-d — CHANGES_REQUESTED** [review]

**Comment by rattandeep1998** [comment]
Thanks for the catch @senali-d 😄

**Review by senali-d — COMMENTED** [review]
Look Good 👍

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍 

Thanks everyone for collaborating 👍

**Pull request merged**